benchexec/tools/cmaesfuzz.py

```diff
@@ -0,0 +1,42 @@
+# This file is part of BenchExec, a framework for reliable benchmarking:
+# https://github.com/sosy-lab/benchexec
+#
+# SPDX-FileCopyrightText: 2007-2020 Dirk Beyer <https://www.sosy-lab.org>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+
+import re
+import benchexec.tools.template
+
+
+class Tool(benchexec.tools.template.BaseTool2):
+    """
+    Fuzzing with stochastic optimization guided by CMA-ES
+
+    Hynsung Kim, Gidon Ernst
+    https://github.com/lazygrey/fuzzing_with_cmaes
+    """
+
+    REQUIRED_PATHS = [
+        "fuzzer",
+        "fuzzer.py",
+        "cma",
+        "verifiers_bytes",
+        "verifiers_real",
+    ]
+
+    def executable(self, tool_locator):
+        return tool_locator.find_executable("fuzzer")
```

Comment:
Is `fuzzer` correct? (Most tools who have files name `tool` and `tool.py` need the wrapper script here.)

Reply:
Yes this is correct, the name came relatively late